Cargo.toml

```diff
@@ -116,8 +116,8 @@ goauth = { version = "0.8.1", optional = true }
 smpl_jwt = { version = "0.5.0", optional = true }
 
 # API
-async-graphql = { version = "2.4.10", optional = true }
-async-graphql-warp = { version = "2.4.10", optional = true }
+async-graphql = { version = "=2.4.11", optional = true }
```

Comment:
This one-liner took a surprisingly long time to figure out :-)

The 2.5 branch of async-graphql bumps to Warp 0.3, which changes the signature of Warp filters and interacting with the GraphQL pieces. The bump was needed because of an issue with `Context` I was running into (the mechanism used to pass the event inspector to the active HTTP request without resorting to a global; I'm trying to move away from the latter.)

Unfortunately, we have other code relying on 0.2.5, which broke a lot of things. After some trial/error, it seems this particular patch version is both stapled to the right Warp version for us, _and_ fixes the Context issue.

I'd like to follow-up with a general bump to Warp 0.3 later to avoid being stuck on this rung of async-graphql (and also because it's the latest anyway.)

Reply:
#6383 👍